big_tests/tests/gdpr_SUITE.erl

```diff
@@ -196,16 +202,18 @@ pick_backend_for_mam() ->
 mam_required_modules(Backend) ->
     [{mod_mam_meta, [{backend, Backend}, {pm, []}]}].
 
-pick_backend_for_vcard() ->
+pick_backend_for_vcard_or_offline() ->
```

Comment:
Can we rename it as `pick_backend_as_riak_or_rdbms` ? it fits better what it does (there is nothing specific aboud vcard or offline in the function itself).